content/en/continuous_integration/intelligent_test_runner/javascript.md

```diff
@@ -71,13 +71,19 @@ NODE_OPTIONS="-r dd-trace/ci/init" DD_ENV=ci DD_SERVICE=my-javascript-app DD_CIV
 For Intelligent Test Runner for Cypress to work, you must instrument your web application with code coverage. You can read more about enabling code coverage in the [Cypress documentation][3]. To check that you've successfully enabled code coverage, navigate to your web app with Cypress and check the global variable `window.__coverage__`. This is what `dd-trace` uses to collect code coverage for Intelligent Test Runner.
 
 
-#### Suite skipping
+## Suite skipping
 Intelligent Test Runner for Javascript skips entire _test suites_ (test files) rather than individual tests.
 
+
+## Note about changing test durations
```

Comment:
Is this note directly related to the section above (Suite skipping)? 
- If so, I would recommend putting this content in an alert box inside that section instead of introducing a new section. Alternatively, using an H3 and renaming the section to remove the word "Note".
- If not, let's rephrase the section title to remove the word "Note" - it's not consistent with the way we present notes in the docs. Perhaps something like: Inconsistent test durations.

Reply:
it's not. I'll do the change in the title

Reply:
done in https://github.com/DataDog/documentation/pull/19472/commits/1bbdcfe09867fae6707235600e2d8e220ddd08b2